### PR TITLE
Use __dir__ instead of getting the dir of __FILE__

### PR DESCRIPTION
**Pull request by tas50**
This is a bit easier to read

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by phiggins** [comment]
Looks like this is also running into this issue: https://bugs.ruby-lang.org/issues/13126

**Comment by tas50** [comment]
Time to update that chefstyle rule!

**Review by mwrock — APPROVED** [review]

**Comment by lamont-granquist** [comment]
for some reason "the dir of __FILE__" sounds like a royal title in my brain for some reason.

**Review by lamont-granquist — APPROVED** [review]

File: tasks/rspec.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@
 require "rubygems"
 require "rake"
 
-CHEF_ROOT = File.join(File.dirname(__FILE__), "..")
+CHEF_ROOT = File.join(__dir__, "..")
```
Comment (phiggins) [inline tasks/rspec.rb:24]:
```suggestion
```

This isn't referenced anywhere.

**Review by phiggins — APPROVED** [review]

**Pull request merged**